docs/user/build-default-versions.rst

```diff
@@ -0,0 +1,106 @@
+:orphan:
+
+
+.. This page contains good detailed content about the exact versions Read the
```

Comment:
This mostly makes sense. It is very specific information that most users don't need.